fe/fe-core/src/main/java/org/apache/doris/qe/AutoCloseConnectContext.java

```diff
@@ -29,6 +29,9 @@ public AutoCloseConnectContext(ConnectContext connectContext) {
         connectContext.setThreadLocalInfo();
     }
 
+    public void call() {
```

Comment:
useless method?

Reply:
```try (AutoCloseConnectContext autoCloseCtx = new AutoCloseConnectContext(context)) {```
will report autoCloseCtx is not used, so call an empty method.